platform/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@pipedream/platform",
+  "version": "0.11.0",
```

Comment:
@andrewjschuang Thanks for making those changes!

I suggested incrementing the major or minor version in a prior comment. Could we actually increment the major version (to `1.0.0`) to decrease the potential impact of these changes?

Reply:
After merging this PR, could you do some smoke testing with some components that use `@pipedream/platform` to check that everything works as expected?

Reply:
I'll test some before merging too